--- a/configure.ac
+++ b/configure.ac
@@ -32,7 +32,7 @@ AC_ARG_WITH([rst2man],
 		[--with-rst2man=PATH],
 		[Location of rst2man (auto)]),
 	[RST2MAN="$withval"],
-	AC_CHECK_PROGS(RST2MAN, [rst2man rst2man.py], []))
+	[AC_CHECK_PROGS(RST2MAN, [rst2man rst2man.py])])
 AM_CONDITIONAL(HAVE_RST2MAN, [test "x$RST2MAN" != "xno"])
 
 AC_ARG_WITH([lcov],
@@ -40,7 +40,7 @@ AC_ARG_WITH([lcov],
 		[--with-lcov=PATH],
 		[Location of lcov to generate coverage data (auto)]),
 	[LCOV="$withval"],
-	AC_CHECK_PROGS(LCOV, [lcov], []))
+	[AC_CHECK_PROGS(LCOV, [lcov])])
 AM_CONDITIONAL(HAVE_LCOV, [test -n "$LCOV"])
 
 AC_ARG_WITH([genhtml],
@@ -48,7 +48,7 @@ AC_ARG_WITH([genhtml],
 		[--with-genhtml=PATH],
 		[Location of genhtml to generate coverage reports (auto)]),
 	[GENHTML="$withval"],
-	AC_CHECK_PROGS(GENHTML, [genhtml], []))
+	[AC_CHECK_PROGS(GENHTML, [genhtml])])
 AM_CONDITIONAL(HAVE_GENHTML, [test -n "$GENHTML"])
 
 m4_ifndef([VARNISH_PREREQ], AC_MSG_ERROR([Need varnish.m4 -- see README.rst]))
